rules/data_binding.bzl

```diff
@@ -54,7 +54,7 @@ def _copy_annotation_file(ctx, output_dir, annotation_template):
     annotation_out = ctx.actions.declare_file(
         output_dir + "/android/databinding/layouts/DataBindingInfo.java",
     )
-    _utils.copy_file(ctx, annotation_template, annotation_out)
+    ctx.actions.symlink(output = annotation_out, target_file = annotation_template)
```

Comment:
If this is part of your other PR for fixing copy_file() for Mac, let me take another look there and see what we can do to fix things.

Reply:
My other PR doesn't fix copy_file for macOS; it just adds an example to surface that it is broken on macOS. This is where I actually incorporated the fix.